README.md

```diff
@@ -9,6 +9,14 @@ The synchronization works like this
 * creating a new OpenShift Build for a BuildConfig associated with a Jenkins Job results in the Jenkins Job being triggered
 * changes in a Jenkins Build Run thats associated with a Jenkins Job gets replicated to an OpenShift Build object (which is created if necessary if the build was triggered via Jenkins)
 
+Configuration
+------------------------
+Jenkins Build Log URL: 
+* Plugin adds a annotation in Build resource metadata which contains URL to access Jenkins build logs.
+By default host name for log URL comes from service running in OpenShift knows as "Jenkins". To override and configure this URL's host name, add an environment variable `JENKINS_ROOT_URL`. 
```

Comment:
By default host name for log URL -> By default, the Jenkins base URL for the build log is determined via the OpenShift route of the _jenkins_ service. To override and configure this base URL, you can set the environment variable `JENKINS_ROOT_URL`.

Reply:
host name != (base) URL. What are we setting here? I take it is the base URL, right? Then please don't mix this up with hostname. These are two completely different things.

Reply:
+1 for the docs, just needs some rework

Reply:
yes its base URL